workflows/config/snakemake/production-config.yml

```diff
@@ -12,25 +12,36 @@
     "num_samples_per_population": [100,100,100],
     }
 ]
-"genetic_map": "HapMapII_GRCh37"
-"chrm_list": "chr21"
-"dfe_list": ["Gamma_H17"]
-"annotation_list": ["all_sites", "ensembl_havana_104_exons"]
+"genetic_map": "HapMapII_GRCh38"
+"chrm_list": "chr1"
+# "dfe_list": ["Gamma_H17"]
+"dfe_list": ["none"]
+# "annotation_list": ["ensembl_havana_104_exons"]
+"annotation_list": ["none"]
```

Comment:
see comment above-- seems like rather than edit this file you should create a whole new yaml config

Reply:
yeah, we don't want to have "none" in `production-config.yml` as this file will be what specifies the large run for the paper